src/modules/rules.js

```diff
@@ -443,6 +444,45 @@ const RULES = [
     metadata: (version) => ({
       compass_version: version
     })
+  },
+  {
+    registryEvent: 'compass:compass-shell:opened',
+    resource: 'Shell',
+    action: 'opened',
+    condition: () => true,
+    metadata: (version) => ({
+      compass_version: version
+    })
+  },
+  {
+    registryEvent: 'mongosh:show',
+    resource: 'Shell',
+    action: 'show',
+    condition: () => true,
+    metadata: (version, data) => ({
+      compass_version: version,
+      properties: { method: data.method }
+    })
+  },
+  {
+    registryEvent: 'mongosh:use',
+    resource: 'Shell',
+    action: 'use',
+    condition: () => true,
+    metadata: (version) => ({
+      compass_version: version
+    })
+  },
+  {
+    registryEvent: 'mongosh:api-call',
```

Comment:
There is also a `mongosh:error` event which is useful to track, especially for `Mongosh` specific errors.

Reply:
@lrlna Ah good call, thanks for catching. Added.